### PR TITLE
[NetAppFiles] Update to 2022-09-01 

### PR DESCRIPTION
**Pull request by audunn**
## Checklist

- [x] Check this box to confirm: **I have read the [_Submitting Changes_](../blob/main/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/main/CONTRIBUTING.md) and reviewed the following information:**

* **SHOULD** select appropriate branch. Cmdlets from Autorest.PowerShell should go to [`generation`](https://github.com/Azure/azure-powershell/tree/generation) branch. 
* **SHOULD** make the title of PR clear and informative, and in the present imperative tense. 
* **SHOULD** update `ChangeLog.md` file(s) appropriately
    * For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    * A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header in the past tense. Add changelog in description section if PR goes into [`generation`](https://github.com/Azure/azure-powershell/tree/generation) branch.
    * Should **not** change `ChangeLog.md` if no new release is required, such as fixing test case only.
* **SHOULD** have approved design review for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) ([_Microsoft internal only_](../blob/main/CONTRIBUTING.md#onboarding)) with following situations
    * Create new module from scratch
    * Create new resource types which are not easy to conform to [Azure PowerShell Design Guidelines](../blob/main/documentation/development-docs/design-guidelines)
    * Create new resource type which name doesn't use module name as prefix
    * Have design question before implementation
* **SHOULD** regenerate markdown help files if there is cmdlet API change. [Instruction](../blob/main/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)
* **SHOULD** have proper test coverage for changes in pull request.
* **SHOULD NOT** introduce [breaking changes](../blob/main/documentation/breaking-changes/breaking-changes-definition.md) in Az minor release except preview version.
* **SHOULD NOT** adjust version of module manually in pull request

### CODE REVIEW

**Comment by audunn** [comment]
> Hi @audunn , `Unlock-AzNetAppFilesVolumeFileLocks` uses the noun `AzNetAppFilesVolumeFileLocks`, which does not follow the enforced naming convention of using a singular noun for a cmdlet name. Please consider using a singular noun for the cmdlet.name.

Updated/changed

File: src/NetAppFiles/NetAppFiles/Az.NetAppFiles.psd1 [hunk 1]
```diff
@@ -92,17 +92,17 @@ CmdletsToExport = 'Get-AzNetAppFilesAccount', 'New-AzNetAppFilesAccount',
...
+               'New-AzNetAppFilesSnapshotPolicy',  'Set-AzNetAppFilesSnapshotPolicy', 
                'Get-AzNetAppFilesSnapshotPolicy', 
                'Update-AzNetAppFilesSnapshotPolicy', 
                'Remove-AzNetAppFilesSnapshotPolicy', 
                'New-AzNetAppFilesBackupPolicy', 'Get-AzNetAppFilesBackupPolicy', 
                'Set-AzNetAppFilesBackupPolicy', 'Update-AzNetAppFilesBackupPolicy', 
                'Remove-AzNetAppFilesBackupPolicy', 'New-AzNetAppFilesBackup', 
                'Get-AzNetAppFilesBackup', 'Update-AzNetAppFilesBackup', 
-               'Remove-AzNetAppFilesBackup', 'Get-AzNetAppFilesVolumeBackupStatus', 
-               'Get-AzNetAppFilesVolumeRestoreStatus', 'Get-AzNetAppFilesVault',
```
Comment (wyunchi-ms) [inline src/NetAppFiles/NetAppFiles/Az.NetAppFiles.psd1:105]: Looks you remove `Get-AzNetAppFilesVault`. You can also remove the source file `GetAzureRmNetAppFilesVault.cs`

File: src/NetAppFiles/NetAppFiles/Vaults/GetNetAppFilesVault.cs
Comment (wyunchi-ms) [inline src/NetAppFiles/NetAppFiles/Vaults/GetNetAppFilesVault.cs:1]: I think you can remove this file directly.

File: src/NetAppFiles/NetAppFiles/Az.NetAppFiles.psd1 (hunk 1 above)
Comment (audunn) [reply src/NetAppFiles/NetAppFiles/Az.NetAppFiles.psd1:105]: Removed

**Review by wyunchi-ms — APPROVED** [review]

**Pull request merged**